src/controller/controller.go

```diff
@@ -1,36 +1,39 @@
 package controller
 
 import (
+	"reflect"
 	"time"
 
 	"github.com/golang/glog"
-	"k8s.io/api/core/v1"
-	corev1_api "k8s.io/api/core/v1"
+	"github.com/xing/kubernetes-oom-event-generator/src/util"
+	core "k8s.io/api/core/v1"
 	"k8s.io/client-go/informers"
-	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
 	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
-	"github.com/xing/kubernetes-oom-event-generator/src/util"
 )
 
 const (
 	// informerSyncMinute defines how often the cache is synced from Kubernetes
-	informerSyncMinute = 3
```

Comment:
The README needs some adaptation and we should probably mention that value in it

Reply:
Oh yes good point. I'll adjust it.